astronomer/providers/microsoft/azure/example_dags/example_adf_run_pipeline.py

```diff
@@ -23,6 +23,7 @@
 TENANT_ID = os.getenv("TENANT_ID", "")
 SUBSCRIPTION_ID = os.getenv("SUBSCRIPTION_ID", "")
 RESOURCE_GROUP_NAME = os.getenv("RESOURCE_GROUP_NAME", "team_provider_resource_group_test_1")
+ADF_FACTORY_NAME = os.getenv("ADF_FACTORY_NAME", "ADFProvidersTeamDataFactoryTest")
```

Comment:
We already have a variable below `DATAFACTORY_NAME` (which is also set in our Astro Deployments), perhaps we could just use that in the `default_args` dictionary below.

Reply:
@pankajkoti fixed